scalafmt-core/shared/src/main/scala/org/scalafmt/config/ContinuationIndent.scala

```diff
@@ -4,18 +4,29 @@ import metaconfig._
 
 /**
   * @param defnSite indentation around class/def
+  * @param declSite indentation around type/method declarations
+  * @param ctorSite indentation around class constructor parameters
   * @param callSite indentation around function calls, etc.
   * @param extendSite indentation before `extends`
   * @param withSiteRelativeToExtends additional indentation before `with`
   */
 case class ContinuationIndent(
     callSite: Int = 2,
     defnSite: Int = 4,
+    declSite: Option[Int] = None,
+    ctorSite: Option[Int] = None,
     extendSite: Int = 4,
     withSiteRelativeToExtends: Int = 0
 ) {
   implicit val reader: ConfDecoder[ContinuationIndent] =
     generic.deriveDecoder(this).noTypos
+
+  def getDefnSite(tree: meta.Tree): Int =
+    (tree match {
+      case _: meta.Ctor => ctorSite
+      case _: meta.Decl => declSite
```

Comment:
In what situation do we want to distinguish declarations from definitions?

Reply:
frankly, i have no idea. heard someone mention ability to distinguish abstract and concrete methods visually. also, possibly because i don't want to come back to this later, although that's questionable reason.

Reply:
I would avoid introducing the decl/defn separation since it's not self-explanatory to most users what the difference is. The original ticket specifically was about separating class constructors from methods, which seems reasonable

Reply:
all right, will keep `ctor` only.